### DISCUSSION TITLE
Audit functionality for Web-based terminal

### DISCUSSION THREAD

**Discussion by msobkowiak-olx** [argoproj/argo-cd#19604] [General]
Hey there ! Does ArgoCD in current version (2.12.1) offer audit log on the commands executed using the web terminal functionality ?

**Comment by crenshaw-dev** [comment]
Argo CD doesn't not currently support audit logs for web terminal commands. It's trickier to implement than is immediately obvious. https://github.com/argoproj/argo-cd/issues/9918
